### PR TITLE
Updates to duckdb setup groupby and join scripts

### PR DESCRIPTION
**Pull request by Tmonster**

### CODE REVIEW

File: duckdb-latest/setup-duckdb-latest.sh [hunk 1]
```diff
@@ -5,12 +5,12 @@ set -e
 mkdir -p ./duckdb-latest/r-duckdb-latest
 # Rscript -e  'withr::with_libpaths(new = "./duckdb-latest/r-duckdb-latest", devtools::install_github("duckdb/duckdb/tools/rpkg"))'
 # prevent errors when running 'ver-duckdb.sh'
-Rscript -e 'install.packages("DBI", lib="./duckdb-latest/r-duckdb-latest")'
+Rscript -e 'install.packages("DBI", lib="./duckdb-latest/r-duckdb-latest", repos = "http://cran.us.r-project.org")'
```
Comment (jangorecki) [inline duckdb-latest/setup-duckdb-latest.sh:8]: it is generally better to use `cloud.r-project.org` so other regions will not have to via us mirror

File: duckdb/join-duckdb.R [hunk 2]
```diff
@@ -5,13 +5,13 @@ cat("# join-duckdb.R\n")
 source("./_helpers/helpers.R")
 
 suppressPackageStartupMessages({
-  library("DBI", lib.loc="./duckdb/r-duckdb", warn.conflicts=FALSE)
-  library("duckdb", lib.loc="./duckdb/r-duckdb", warn.conflicts=FALSE)
+  library("DBI", lib.loc="./duckdb-latest/r-duckdb-latest", warn.conflicts=FALSE)
```
Comment (jangorecki) [inline duckdb/join-duckdb.R:8]: there is a separate directory storing test scripts for duckdb-latest, any reason why duckdb (not latest) will be using latest repo?
unless it is not going into master branch...

File: duckdb/join-duckdb.R [hunk 3]
```diff
@@ -61,22 +63,31 @@ invisible({
...
 if (!uses_NAs) {
+  if (on_disk) {
+    unlink('clean.db')
+    invisible(dbExecute(con, "attach 'clean.db'"))
+    db_name = "clean."
+  }
+  else {
+    db_name = ""
+  }
+
```
Comment (jangorecki) [inline duckdb/join-duckdb.R:74]: I wonder why it is nested in `!uses_NA` branch?

File: duckdb/join-duckdb.R (hunk 2 above)
Comment (Tmonster) [reply duckdb/join-duckdb.R:8]: For this run we will have a duckdb 0.9.0 and duckdb 0.8.1-3 version in the report to show the improvement. Eventually we will show just one version

File: duckdb/join-duckdb.R (hunk 3 above)
Comment (Tmonster) [reply duckdb/join-duckdb.R:74]: If there are no NA's, then there is an extra step during ingestion where we attach a new persistent db that has just the tables with enums. If there are NA's we just rename the tables and don't need to special logic. 

If I'm not mistaken duckdb can handle NA's in enums. I will update the script to reflect this when the report gets published

File: duckdb/join-duckdb.R (hunk 2 above)
Comment (Tmonster) [reply duckdb/join-duckdb.R:8]: sorry, responded to quickly. This should just be going to `duckdb`

**Pull request merged**